src/app/static/src/js/report.js

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue';
 import $ from 'jquery';
```

Comment:
Is this code base quite old, hence the jquery?

Reply:
No it's about 2 years old. We use jquery in a few places although you're right that in this case it could easily be done with vanilla js.

Unlike hint, this app isn't a vuex app, but a regular MVC app with most content rendered on the server using freemarker templating - vue is sometimes used when necessary, e.g. for a dynamic widget on a page like this one, but we also use bootstrap, jquery and [datatables](https://datatables.net/) where appropriate.

Reply:
I see. I knew there were a few edge cases for using jquery but I know very little about it so I thought I'd ask.

Reply:
This is interesting, were you taught that jquery is dead? Outside of single page apps it's still very widely used.

Reply:
Not dead as it's still widely used as you say, just that newer apps tend not to use it as other techs can generally do what it does better.

Reply:
Also that many of the problems it was built to solve have since been covered by vanilla JS.